infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -504,6 +504,151 @@ all the operations for the connector:
   - `PSOXY_DROPBOX_BUSINESS_CLIENT_ID` with `App key` value.
   - `PSOXY_DROPBOX_BUSINESS_CLIENT_SECRET` with `App secret` value.
 
+EOT
+    },
+    jira-server = {
+      source_kind : "jira-server"
+      worklytics_connector_id : "jira-server-psoxy"
+      target_host : var.jira_server_url
+      source_auth_strategy : "oauth2_access_token"
+      display_name : "Jira REST API"
+      identifier_scope_id : "jira"
+      worklytics_connector_name : "Jira Server REST API via Psoxy"
+      secured_variables : [
+        { name : "ACCESS_TOKEN", writable : false },
+      ],
+      reserved_concurrent_executions : null
+      example_api_calls_user_to_impersonate : null
+      example_api_calls : [
+        "/rest/api/2/search?maxResults=25",
+        "/rest/api/2/issue/${var.example_jira_issue_id}/comment?maxResults=25",
+        "/rest/api/2/issue/${var.example_jira_issue_id}/worklog?maxResults=25",
+        "/rest/api/latest/search?maxResults=25",
+        "/rest/api/latest/issue/${var.example_jira_issue_id}/comment?maxResults=25",
+        "/rest/api/latest/issue/${var.example_jira_issue_id}/worklog?maxResults=25",
+      ],
+      external_token_todo : <<EOT
+1. Follow the instructions to create a [Personal Access Token](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html) in your instance
+2. Disable or mark a proper expiration of the token.
+3. Copy the value of the token in PSOXY_JIRA_SERVER_ACCESS_TOKEN variable as part of AWS System Manager parameters store / GCP Cloud Secrets (if default implementation)
+NOTE: If your token has been created with expiration date, please remember to update it before that date to ensure connector is going to work.
+EOT
+    }
+    jira-cloud = {
+      source_kind : "jira-cloud"
+      worklytics_connector_id : "jira-cloud-psoxy"
+      target_host : "api.atlassian.com"
+      source_auth_strategy : "oauth2_refresh_token"
+      display_name : "Jira REST API"
+      identifier_scope_id : "jira"
+      worklytics_connector_name : "Jira REST API via Psoxy"
+      secured_variables : [
+        { name : "ACCESS_TOKEN", writable : true },
+        { name : "REFRESH_TOKEN", writable : true },
+        { name : "CLIENT_ID", writable : false },
+        { name : "CLIENT_SECRET", writable : false },
+        { name : "OAUTH_REFRESH_TOKEN", writable : true }, # Lock for sharing access token, handled by Psoxy
+      ],
+      environment_variables : {
+        GRANT_TYPE : "refresh_token"
+        REFRESH_ENDPOINT : "https://auth.atlassian.com/oauth/token"
+        USE_SHARED_TOKEN : "TRUE"
```

Comment:
why isn't this being set for `zoom` as well?

Reply:
Now required as `HSE_SHARED_TOKEN`  property is used for GCP to create the secret, but in `account_credentials` grant type (the one Zoom uses) that value is always hardcoded to true.